packages/config/src/configDefaults.ts

```diff
@@ -33,6 +33,11 @@ export const getInitialConfig = ({
     build: null,
     resolver: null,
     artifactor: null,
+    dashboard: {
+      host: '0.0.0.0',
```

Comment:
Just a drive by observation that `0.00.0` would bind to all resolved network addresses on a multi homed computer if used

Reply:
Hmm right. Do you think that would be a problem, would it be better to just default to `localhost` then and allow someone to specify 0.0.0.0 if they want?

Reply:
I think defaulting to `localhost` and documenting the options for users to make an educated choice is better all around.

Reply:
Either way is fine by me. Anyone else have a different opinion? Otherwise we'll go with your suggestion Amal.